pom.xml

```diff
@@ -101,7 +101,7 @@
     <revision>2.106</revision>
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/email-ext-plugin</gitHubRepo>
-    <jenkins.version>2.387.3</jenkins.version>
+    <jenkins.version>2.440.1</jenkins.version>
```

Comment:
needed to update to get the FIPS support from core.  Chose the latest LTS to maintain a healthy plugin.